### PR TITLE
Code refactoring in main.js

### PR DESCRIPTION
**Pull request by demvlad**
Some code refactoring in main.js.
Removed duplicated function call in main.js.
The let is change to const when it was need.

### CODE REVIEW

**Review by nerdCopter — APPROVED** [review]
- tested good

File: src/main.js [hunk 1]
```diff
@@ -2170,25 +2162,6 @@ function BlackboxLogViewer() {
...
-        prefs.get('activeWorkspace', function (id){
-            if (id) {
-                activeWorkspace = id
-            }
-            else {
-                activeWorkspace = 1
-            }
-        });
-        onSwitchWorkspace(workspaceGraphConfigs, activeWorkspace);
-
```
Comment (haslinghuis) [inline src/main.js:2191]: This is handled somewhere else or no longer needed?

Comment (nerdCopter) [reply src/main.js:2191]: src/main.js:1072

Comment (demvlad) [reply src/main.js:2191]: Yes this code exists above

**Comment by demvlad** [comment]
It was interst
This code works here and does not work in NW.js,
because NW.js call (3) and next code below first, and (1), (2)  the most lastest. And there are not asinchrone paralell code.
I was some confused :)

prefs.get('workspaceGraphConfigs', function(item) {
(1)
            if (item) {
                workspaceGraphConfigs = upgradeWorkspaceFormat(item);
            } else {
                workspaceGraphConfigs = workspaceMenu.getDefaultWorkspace();
            }
        });

        prefs.get('activeWorkspace', function (id){
(2)
            if (id) {
                activeWorkspace = id;
            }
            else {
                activeWorkspace = 1;
            }
        });
(3)
        workspaceSelection = new WorkspaceSelection($(".log-workspace-selection"), workspaceGraphConfigs, 
        onSwitchWorkspace, onSaveWorkspace);
        onSwitchWorkspace(workspaceGraphConfigs, activeWorkspace);

File: src/main.js (hunk 1 above)
Comment (haslinghuis) [reply src/main.js:2191]: Yes was lazy to search for myself :see_no_evil:

Comment (demvlad) [reply src/main.js:2191]: Yes this code exists above

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**